pages/common/composer.md

```diff
@@ -0,0 +1,31 @@
+# composer
+
+> Dependency Manager for PHP.
+
+- Interactively create a composer.json file defining the project's dependencies:
+
+`composer init`
+
+- Create a new project from a given version of an existing package:
```

Comment:
@agnivade do [these comments](https://github.com/tldr-pages/tldr/pull/975#r71986760) shed any light into this?